pom.xml

```diff
@@ -164,7 +164,22 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+
+			<plugin>
+				<groupId>org.flywaydb</groupId>
```

Comment:
we already have a flyway dependency in the pom.xml

Reply:
Kindly check and confirm if there is a  flyway dependency in the pom.xml, When I cloned it was not present, I checked again now and it is still not present.

![Screenshot 2024-07-24 at 8 59 06 PM](https://github.com/user-attachments/assets/6c00e972-9956-49b8-b4b9-56ee3c58d99b)

Reply:
Kindly check and confirm if there is a  flyway dependency in the pom.xml, When I cloned it was not present, I checked again now and it is still not present.

![Screenshot 2024-07-24 at 8 59 06 PM](https://github.com/user-attachments/assets/6c00e972-9956-49b8-b4b9-56ee3c58d99b)